src/logger.js

```diff
@@ -19,3 +19,25 @@ export const getLogger: GetLogger = memoize(() => {
     url: loggerUrl,
   });
 });
+
+export const sendCountMetric = ({
+  dimensions,
+  event = "unused",
```

Comment:
we should make event required, i was wrong when i initially set these as `unused`. It makes the queries for metrics really long if they don't have an event to filter on

Reply:
Makes sense, checking to see if there are any being used without event